### PR TITLE
Update API version from v1beta1 to v1 for Netapp Trident templates

### PR DESCRIPTION
**Pull request by anandkarra**
Updated Netapp Trident templates with new revisions to update API version from `v1beta1` to `v1` for `PodSecurityPolicy`.

### CODE REVIEW

File: config-templates/netapp/netapp-trident/changelog.json [hunk 1]
```diff
@@ -3,11 +3,22 @@
         "22.04": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "13 January 2023",
```
Comment (mssachan) [inline config-templates/netapp/netapp-trident/changelog.json]: Change date to `16 jan`

File: config-templates/netapp/netapp-trident/changelog.json [hunk 2]
```diff
@@ -3,11 +3,22 @@
...
+                "cves": "",
+                "golang": "1.18.8",
```
Comment (mssachan) [inline config-templates/netapp/netapp-trident/changelog.json]: This image is not built by us so please keep `golang` & `ubi` fields empty

File: config-templates/netapp/netapp-trident/changelog.json [hunk 3]
```diff
@@ -3,11 +3,22 @@
...
+                ]
+            },
+            "previous-revision": {
                 "revision": "2",
                 "release-date": "18 July 2022",
                 "cves": "",
-                "golang": "",
-                "ubi": "",
+                "golang": "1.18.8",
+                "ubi": "8.7-923.1669829893",
```
Comment (mssachan) [inline config-templates/netapp/netapp-trident/changelog.json]: same here

**Review by mssachan — CHANGES_REQUESTED** [review]

File: config-templates/netapp/netapp-trident/changelog.json (hunk 3 above)
Comment (anandkarra) [reply config-templates/netapp/netapp-trident/changelog.json]: Addressed in https://github.com/IBM/ibm-satellite-storage/pull/266/commits/b8f8c2e3ff4caac6b4a2c99e1b2d31fc64896345.

File: config-templates/netapp/netapp-trident/changelog.json (hunk 2 above)
Comment (anandkarra) [reply config-templates/netapp/netapp-trident/changelog.json]: Addressed in https://github.com/IBM/ibm-satellite-storage/pull/266/commits/b8f8c2e3ff4caac6b4a2c99e1b2d31fc64896345.

File: config-templates/netapp/netapp-trident/changelog.json (hunk 1 above)
Comment (anandkarra) [reply config-templates/netapp/netapp-trident/changelog.json]: Addressed in https://github.com/IBM/ibm-satellite-storage/pull/266/commits/b8f8c2e3ff4caac6b4a2c99e1b2d31fc64896345.

**Review by mssachan — APPROVED** [review]
lgtm

**Pull request merged**